app/models/recommendation.py

```diff
@@ -98,16 +98,17 @@ async def get_recommendations_from_experiment(
 
         # apply rankers from the slate experiment on the candidate set's candidates
         for ranker in experiment.rankers:
-            if ranker == 'thompson-sampling':
+            if ranker in THOMPSON_SAMPLING_RANKERS:
```

Comment:
Mentioned this to Matt already, but I'd love to spend a fun Friday to see if we can configure the metadata directly on the ranker using a function decorator. That would save us from having to configure that in the __init__.py, and we wouldn't need to make changes when we add a simple `partial` function. :crossed_fingers: